frontend/my-app/src/Pages/Account.tsx

```diff
@@ -1,97 +1,195 @@
 import React, { useState } from 'react'
-import './Account.css'
 import { useLocation, useNavigate } from 'react-router-dom'
+import { AppBar, Box, Button, IconButton, Paper, Stack, TextField, Toolbar, Typography } from '@mui/material'
+
+const stackStyles = {
+  backgroundColor: 'secondary.main',
+  p: 1,
+  justifyContent: 'space-evenly',
+  alignItems: 'center'
+}
 
 const Account: React.FC = () => {
+  const NEW_EMAIL = 'New Email'
+  const NEW_ADDRESS = 'New Address'
+  const NEW_PASSWORD = 'New Password'
+  const NEW_CONFIRM_PASSWORD = 'Confirm New Password'
+
   const location = useLocation()
   const navigate = useNavigate()
 
   const [email, setEmail] = useState<string>(location.state.email)
-  const [userLocation, setLocation] = useState<string>(location.state.location)
-  const [password, setPassword] = useState<string>('')
-  const [passwordConfirm, setPasswordConfirm] = useState<string>('')
+  const [newEmail, setNewEmail] = useState<string>(NEW_EMAIL)
+  const [address, setAddress] = useState<string>(location.state.location)
+  const [newAddress, setNewAddress] = useState<string>(NEW_ADDRESS)
+  const [newPassword, setNewPassword] = useState<string>(NEW_PASSWORD)
+  const [confirmPassword, setConfirmPassword] = useState<string>(NEW_CONFIRM_PASSWORD)
 
   const handleDeleteAccount = (): void => {
-    console.log('Delete Account dummy function')
+    console.log('Delete Account dummy function (STRETCH GOAL)')
```

Comment:
Might want to add TODOs to delete these logs later (they're also stretch goals so idk how important it is to do so lol)